nntrainer/src/bn_layer.cpp

```diff
@@ -34,6 +34,20 @@ namespace nntrainer {
 
 enum class BNParams { mu, var, gamma, beta };
 
+BatchNormalizationLayer &BatchNormalizationLayer::
+operator=(BatchNormalizationLayer &&rhs) noexcept {
```

Comment:
Do we need to create this move constructor? In my knowledge, it is recommended to not write a copy/move constructor if it is the same as the default generated one (which I think in this case and below cases is).

Reply:
In order to pass the coverity issues, we may have to specify explicitly I guess.